feat: E2E testing (#780)
aeternity/aescan: README.md

The core browser set targets the 2 most recent major versions of Chrome, Firefox, and Edge on a monthly basis and Safari
on a yearly basis.

## Testing

> This project uses [Applitools Visual AI](https://cypress.io/) for e2e testing.
> Locally can be run by ```cypress open```

Reviewer: ```suggestion
This project uses [Cypress](https://cypress.io/) for e2e testing.
Locally can be run using the command ```yarn e2e:open```.
```

Stay up-to-date on the latest developments in the æternity ecosystem by following us on social media.

- [Forum](https://forum.aeternity.com/)

>   - [X](https://x.com/aeternity)

Reviewer: ```suggestion
- [X](https://x.com/aeternity)
```
